lib/publiccloud/utils.pm

```diff
@@ -933,10 +931,10 @@ sub zypper_call_remote {
             if ($instance->ssh_script_run(qq[sudo grep "Error code.*502" $log]) == 0) {
                 die 'According to bsc#1070851 zypper should automatically retry internally. Bugfix missing for current product?';
             }
-            elsif ($instance->ssh_script_run(qq[sudo grep "Solverrun finished with an ERROR" $log] == 0)) {
-                my $search_conflicts = q[sudo awk 'BEGIN {print "Processing conflicts - ",NR; group=0}
+            elsif ($instance->ssh_script_run(qq[sudo grep "Solverrun finished with an ERROR" $log]) == 0) {
+                my $search_conflicts = q[sudo awk 'BEGIN {print "Processing conflicts - "; group=0}
```

Comment:
See VR in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/24117#issuecomment-3585142047